tests/integration/targets/mount/tasks/main.yml

```diff
@@ -406,3 +420,283 @@
     - /tmp/myfs1
     - /tmp/test_fstab
   when: ansible_system in ('Linux')
+
+######################################################################################################
+######################################################################################################
+######################################################################################################
+
+- name: Block to test ephemeral option
+  environment:
+    PATH: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+  block:
+  - name: Patch centos repolist
+    ansible.builtin.shell:
+      cmd: "{{ item }}"
+    with_items:
+    - sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-*
+    - sed -i 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' /etc/yum.repos.d/CentOS-*
+    when: ansible_distribution in ('CentOS') and ansible_distribution_version|int >= 8
```

Comment:
CentOS8(or later) is no longer supported for the integration tests. So I think you can remove this `Patch centos repolist` task from the integration test.

- https://github.com/ansible-collections/overview/issues/45#issuecomment-944657318

Reply:
Removed in latest push